components/badges/RequirementRow.jsx

```diff
@@ -0,0 +1,168 @@
+var React = require('react');
+
+var RequirementRow = React.createClass({
+  propTypes: {
+    position: React.PropTypes.number.isRequired,
+    text: React.PropTypes.string.isRequired,
+    icon: React.PropTypes.oneOfType([
+      React.PropTypes.bool,
+      React.PropTypes.string
+    ])
+  },
+
+  getInitialState: function() {
+    return {
+      evidenceFiles: [],
+      renderEvidenceFields: false
+    };
+  },
+
+  render: function () {
+    var icon = null;
+
+    if (this.props.icon !== false) {
+      icon = this.props.icon ? this.props.icon : 'fa fa-check';
+    }
+
+    return (
+      <li>
+        <div className="icon-wrapper">
+          <span className={icon}></span>
+        </div>
+        <div className={"requirement" + (this.state.renderEvidenceFields ? " expanded" : "")}>
+          <div className="task-name">{ this.props.text }</div>
+          <div className="evidence">{ this.renderEvidenceOptions()}</div>
+        </div>
+      </li>
+    );
+  },
+
+  renderEvidenceOptions: function() {
+    if (!this.props.evidence) {
+      return null;
+    }
+
+    if (!this.state.renderEvidenceFields) {
+      return (
+        <button className="btn" onClick={() => this.setState({ renderEvidenceFields: true })}>Submit evidence for this task</button>
+      );
+    }
+
+    return (
+      <div className="apply-send-qualifications">
+        {/*
+          <div className="evidence-title">{ this.props.evidence }</div>
```

Comment:
fun fact: uncommenting this in the PR for parsing evidence and criteria as HTML lists, so no need to remove here.